resources/js/src/app/components/common/LazyImg.vue

```diff
@@ -1,23 +1,34 @@
 <template>
-    <img v-if="!isBackgroundImage" :data-src="imageUrl">
-    <div v-else :data-background-image="imageUrl">
+    <picture v-if="!isBackgroundImage" :data-iesrc="fallbackUrl || imageUrl" :data-picture-class="sizingClass">
+        <source :srcset="imageUrl" :type="mimeType">
+    </picture>
```

Comment:
Hier fehlt ein `<img>`-Tag innerhalb des `<picture>`-Tags mit dem Fallback-Bild für browser die webp, oder `<picture>` nicht unterstützen.

Reply:
https://www.html5rocks.com/en/tutorials/responsive/picture-element/#toc-file-type

Reply:
Das <img> Tag wird von lozad dynamisch erzeugt. Wir sind in dem Fall den Usage Guide von Lozad gefolgt, wo drinne erklärt wird, wie man in Verbindung von <picture> mit lozad arbeitet.

https://apoorv.pro/lozad.js/#example-with-picture-tag

Reply:
> for disabled JS you can set `<noscript><img src="images/thumbs/04.jpg" alt=""></noscript>`

Dann sollten wir das evtl. noch ergänzen, um für Bots lesbar zu sein?!

Reply:
Werd ich umsetzen.